include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -664,396 +659,197 @@ namespace picongpu
                 }
 
 
-                /** Write Amplitude data to HDF5 file
+                /** Write Amplitude data to openPMD file
                  *
                  * Arguments:
                  * Amplitude* values - array of complex amplitude values
                  * std::string name - path and beginning of file name to store data to
                  */
-                void writeHDF5file(std::vector<Amplitude>& values, std::string name)
+                void writeOpenPMDfile(std::vector<Amplitude>& values, std::string name)
                 {
-                    splash::SerialDataCollector hdf5DataFile(1);
-                    splash::DataCollector::FileCreationAttr fAttr;
-
-                    splash::DataCollector::initFileCreationAttr(fAttr);
-                    fAttr.enableCompression = compressionOn;
-
                     std::ostringstream filename;
-                    filename << name << currentStep;
-
-                    hdf5DataFile.open(filename.str().c_str(), fAttr);
-
-                    typename PICToSplash<Amplitude::complex_T::type>::type radSplashType;
-
-
-                    splash::Dimensions bufferSize(
-                        Amplitude::numComponents,
-                        radiation_frequencies::N_omega,
-                        parameters::N_observer);
-
-                    splash::Dimensions componentSize(1, radiation_frequencies::N_omega, parameters::N_observer);
-
-                    splash::Dimensions stride(Amplitude::numComponents, 1, 1);
+                    // TODO: needs to be changed to ".h5" and also support adios
+                    filename << name << "%T_0_0_0.h5";
+
+                    ::openPMD::Series openPMDdataFile = ::openPMD::Series(filename.str(), ::openPMD::Access::CREATE);
+                    ::openPMD::Iteration openPMDdataFileIteration = openPMDdataFile.iterations[currentStep];
+                    openPMDdataFile.setMeshesPath(meshesPathName.c_str());
```

Comment:
```suggestion
                    openPMDdataFile.setMeshesPath(meshesPathName);
```
The current version only works because C++ automatically converts it back to `std::string` by using the C-string constructor.